scripts/external_api_test.py

```diff
@@ -66,11 +66,12 @@ def main():
                 if '/' in key:
                     locale = key.split('/', 1).pop()
 
-                    # v2 urls
-                    url = onyx + '/v2/links/fetch/' + locale
-                    if url not in urls:
-                        urls[url] = set()
-                    urls[url].add(value['legacy'])
+                    # v2 urls, only for desktop
+                    if channel_name == 'desktop':
```

Comment:
I can see where we could try to avoid errors with .lower(), however, I'm more incline to leave the code be. Should this be a problem, we could consider making everything lowercase